doc/install_generic.txt

```diff
@@ -0,0 +1,130 @@
+.. include:: css.txt
+
+.. _install_generic:
+
+
+Generic Installation Instructions
+#################################
+
+If you want the latest on the edge installation of Theano, you want `bleeding_edge`_.
+Otherwise go for the `stable`_.
+
+.. note::
+
+   If you are behind a proxy, you must do some extra configuration steps
+   before starting the installation. You must set the environment
+   variable ``http_proxy`` to the proxy address. Using bash this is
+   accomplished with the command
+   ``export http_proxy="http://user:pass@my.site:port/"``
+   You can also provide the ``--proxy=[user:pass@]url:port`` parameter
+   to pip. The ``[user:pass@]`` portion is optional.
+
+
+.. _stable:
+
+Stable Installation
+-------------------
+
+The recommended way to obtain the latest stable version of Theano is from PyPI using
+pip_. Any argument between <...> means it is optional. 
```

Comment:
Move the "Any argument between <...>" part as the first point of the bullet list please.